components/MainLayout.jsx

```diff
@@ -0,0 +1,3 @@
+function MainLayouy() {
+  return (<section></section>);
```

Comment:
Codacy has a fix for the issue: [Replace `(<section></section>)` with `<section></section>`](https://app.codacy.com/gh/thamizhMsys/testRepo6/pullRequest?prid=9650719)
```suggestion
  return <section></section>;
```